src/logic.js

```diff
@@ -23,7 +23,7 @@ define([
             "hintItext",
         ],
         NO_SELF_REFERENCES = _.without(XPATH_REFERENCES, 'constraintAttr'),
-        CASE_REF_ID = "#case/";
+        EXTERNAL_REF = "#";
```

Comment:
This constant (variable) is only ever used as a key in the `forward` and `reverse` mappings. It is not (nor is it intended to be) used to match the beginning of a hashtag or path. It's simply a key that should never collide with a mug ufid, and it's value could be anything that meets that criteria. I figured `#` was good enough for that, but happy to entertain other suggestions.